apps/widget/cypress/e2e/notifications-list.spec.ts

```diff
@@ -171,11 +171,12 @@ describe('Notifications List', function () {
     cy.getByTestId('unseen-count-label').should('contain', '99+');
   });
 
-  it('pagination check', async function () {
+  it('pagination check', function () {
     cy.wait('@getNotificationsFirstPage');
     cy.task('createNotifications', {
       organizationId: this.session.organization._id,
       enumerate: true,
+      ordered: true,
```

Comment:
this test verifies notifications in the order, so I've added the option to create notifications ordered (trigger with a small delay after)